config/locales/views/results/en.yml

```diff
@@ -5,7 +5,7 @@ en:
       across_all_submission_files: Click on the filename to jump to the annotation.
       common: Common Annotations
       include_in_download: Include Annotations
-      include_in_download_warning: 'Warning: Annotations on PDFs are not currently downloaded.'
+      include_in_download_warning: 'Warning: Annotations on PDFs and URLs are not currently downloaded.'
```

Comment:
This isn't necessary here. We haven't enabled any sort of annotations on URLs yet so this warning is misleading. Please revert this change.